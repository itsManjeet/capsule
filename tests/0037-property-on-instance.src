let Foo = fun () {
    let self = {};

    return self;
};

let foo = Foo ();
println (foo.bar = "bar value");
println (foo.barz = "barz value");

println (foo.bar);
println (foo.barz);