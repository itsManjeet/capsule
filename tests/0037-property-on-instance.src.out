{bar:bar value}
{bar:bar value, barz:barz value}
bar value
barz value
